File: Python Word Examples/TextRangesApiDemo/deleteTextRange.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TextRangesApi(configuration)

file_name = 'Template.docx'
paragraph_path = 'Section/0/Body/0/Paragraph/0'
index = 0
extra = {'folder': "input", 'dest_file_path': 'output/DeleteTextRange.docx'}
result = api.delete_text_range(file_name, paragraph_path, index, **extra)

File: JavaScript Word Examples/TextRangeApiDemo/deleteTextRange.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TextRangesApi(apiClient);

    var filename = "DeleteTextRange.docx";
    var paragraphPath = "Section/0/Body/0/Paragraph/0";
    var destFilePath = "output/DeleteTextRange_out.docx";
    var index = 0;
    var opt = {
        folder: 'input',
        storage: '',
        password: ''
    }
    instance.deleteTextRange(filename, paragraphPath, index, destFilePath, opt, 
        function (error, data, response) {
            if (error) throw error;
        });
})();